apps/envited-x-data-space/common/globalIdentifiers/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const SUPPORTED_PATTERNS = {
```

Comment:
Do we need someting with "method" in the name?

Reply:
I dont think so